src/webgl/camera.js

```diff
@@ -34,9 +34,113 @@ var p5 = require('../core/core');
  * blue square shrinks in size grows to fill canvas. disappears then loops.
  *
  */
-p5.prototype.camera = function(x, y, z){
-  //what it manipulates is the model view matrix
-  this._renderer.translate(-x, -y, -z);
+p5.prototype.camera = function(){
+  this._renderer.camera.apply(this._renderer, arguments);
+};
+
+p5.RendererGL.prototype.camera = function () {
+  var eyeX, eyeY, eyeZ;
+  var centerX, centerY, centerZ;
+  var upX, upY, upZ;
+  if (arguments.length === 0) {
+    eyeX = this.defaultCameraX;
+    eyeY = this.defaultCameraY;
+    eyeZ = this.defaultCameraZ;
+    centerX = eyeX;
+    centerY = eyeY;
+    centerZ = 0;
+    upX = 0;
+    upY = 1;
+    upZ = 0;
+  } else {
```

Comment:
looks like the documentation requests three numbers, but this method is expecting 0 or 9. can we update the docs to reflect these two options?

Reply:
does it make most sense to pass in the arguments at 9 numbers, or as three vectors of some kind? maybe plain numbers matches better with the rest of the API?

Reply:
Yes. I've also been wondering if this 9-argument (or 3-vector) version is outdated in 2017 (it didn't seem that three.js still defines the camera in this way, for example, but this is what feels comfortable to me from gluLookAt). It probably makes sense for their to be a 3-argument version, too, or another method called lookAt that would only affect the aim of the camera.